examples/blog/src/view/LoginView.jsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
-import Rx from "rx";
+import Rx from "rxjs";
 import { connect } from "react-redux";
 import wrapActionCreators from "react-redux/lib/utils/wrapActionCreators";
-import { rxConnect, ofActions } from "rx-connect";
+import { rxConnect, ofActions } from "../../../../src";
```

Comment:
Check Webpack's config and `alias` section. `from "rx-connect"` was aliasing to the sources actually

Reply:
Ah cool. Yeah that's much nicer. I was confused because `rx-connect` is also specified as a dependency in the `package.json`: https://github.com/bsideup/rx-connect/blob/master/examples/blog/package.json#L19